### PR TITLE
Adds "tldr" to commit message.

### PR DESCRIPTION
**Pull request by rwaldron**
Example:

$ /scripts/commit.sh "array concat arraylikes"

Import tests from Google V8 (array concat arraylikes)
These tests are derived from the following files within the Google V8 project:

$ /scripts/commit.sh
Import tests from Google V8
These tests are derived from the following files within the Google V8 project:

### CODE REVIEW

File: scripts/commit.sh [hunk 1]
```diff
@@ -14,4 +20,6 @@ $FILES
 EOF
 )
 
-gits commit --message "$MESSAGE" $@
+echo $MESSAGE
+
+# gits commit --message "$MESSAGE" $@
```
Comment (jugglinmike) [inline scripts/commit.sh]: Did you intend for this change to be in the patch?

File: scripts/commit.sh [hunk 2]
```diff
@@ -2,10 +2,16 @@
 
 # Prepare a commit message that includes a listing of files removed from V8
 
+tldr=""
```
Comment (jugglinmike) [inline scripts/commit.sh]: Mind sticking with the CAPS convention for variables?

File: scripts/commit.sh (hunk 1 above)
Comment (rwaldron) [inline scripts/commit.sh]: Nope.

File: scripts/commit.sh (hunk 2 above)
Comment (rwaldron) [inline scripts/commit.sh]: Not at all.

**Comment by rwaldron** [comment]
@jugglinmike fixed

**Comment by jugglinmike** [comment]
Cool!

**Pull request merged**